### ISSUE TITLE
Increase size of icons and buttons

### ISSUE THREAD

**Issue by ventolinmono** [keepassxreboot/keepassxc#3585]
## Summary
I give digital security workshops and some people have commented that the icons and buttons are small, even on normal screens (not hi-dpi).

## Desired Behaviour
Normal and bigger icons and buttons.

## Possible Solution
An option on general settings to make icons and buttons bigger.

**Comment by droidmonkey** [comment]
Can you be more specific please?

**Comment by ventolinmono** [comment]
The icons on the top bar and the icons for each entry are too small for some people, even with an average screen size (1920 - 1080 px).

I was thinking maybe an option on general settings to choose between normal and bigger icons will do.

**Comment by kemalsecer** [comment]
I think this option should also apply to the tree view.

**Comment by wolframroesler** [comment]
#475/#4066 change all icons from fixed-size (PNG) to scalable (SVG), maybe that facilitates making the tool bar size configurable.

**Comment by droidmonkey** [comment]
Yes we can change the toolbar size in the UI file for MainWindow

**Comment by wolframroesler** [comment]
Here's what the toolbar currently looks like with the re-designed icons (as of #475/#4066):

![main](https://user-images.githubusercontent.com/25565229/71765797-de072a00-2ef8-11ea-9212-b428259f26e0.png)

There's some vertical space left due to the height of the search field. How about increasing the icon size from 16x16 to 32x32, and maybe removing some rarely used icons (like Password Generator or Lock Databases) to clean it up a bit? If it looks good then maybe we don't even need the icon size to be configurable.

**Comment by droidmonkey** [comment]
We will definitely catch flak for removing some toolbar icons, like the lock icon. The password generator icon, to me, is about discoverability for new users, there is also not a good keyboard shortcut to open that up (Ctrl+G ??).

I personally think there should be a database settings button to the left of the application settings button. Like the barrel with a cog wheel.

**Comment by phoerious** [comment]
I mentioned that in the other thread. I would remove "New", "Settings", and "Save" when auto-save is enabled. "Lock" should stay, so should the pw generator button.

**Comment by wolframroesler** [comment]
@phoerious wrote in #4066 ("it" refers to the Settings toolbar icon):

> If we don't want to remove it, we could perhaps replace the settings icon with a less obtrusive hamburger icon and place it somewhere else. Can we do CSDs in Qt?

Hamburger means "menu", I wouldn't use it for "settings". The most common icon for "settings" these days is the cogwheel. BTW, to avoid confusion of "settings" and "database settings", I suggest renaming the former to "preferences". ("Application settings" would also do but is too technical IMHO.)

What are CSDs?

**Comment by wolframroesler** [comment]
This is what the toolbar looks like with 32x32 Material Design icons, Save and Settings removed, and a separator between Copy URL and Auto-type. Full window screenshot for better orientation.

![main](https://user-images.githubusercontent.com/25565229/71767881-672a5b00-2f11-11ea-8c47-1d9e17ffe234.png)

Really like the size, looks much better than the 22x22 we had before, and goes much better with the height of the search field. Do we really need to support smaller icons in the tool bar?

**Comment by phoerious** [comment]
I still have a problem with the username, key, and world icons. They are pretty generic representations of exactly those concepts, but none of them really convey the affordance of "copying" something.

**Comment by droidmonkey** [comment]
The Auto-Type icon is also very abstract

**Comment by wolframroesler** [comment]
@phoerious, @droidmonkey Adding alternative suggestions for the icons to #4066.

**Comment by wolframroesler** [comment]
Toolbar icons have been enlarged from 22x22 to 32x32, #4066 has been merged, is there anything left to do here?

**Comment by droidmonkey** [comment]
Nope we are done
